lang/de.json

```diff
@@ -125,6 +125,7 @@
                 "action":                                           "Aktion",
                 "active":                                           "Aktiv",
                 "amount":                                           "Menge",
+                "ammunition":                                       "Munition / Menge",
```

Comment:
It says `ammunition.type` in the localization calls

Reply:
ok fixed the relevant parts